Assets/Scripts/Controllers/CharController.cs

```diff
@@ -8,15 +8,15 @@ public class CharController : MonoBehaviour
     protected new String name = "Character";
     public bool doneTurn;
     private const float Speed = 10f;
-    protected int maxHealth = 100;
```

Comment:
why remove the default value?

Reply:
Because enemies and players will have different max healths set by their controllers.

Reply:
That's true, but it's still good practice to have a default to  fall back on

Reply:
Im going to move everything to use its getters/setters.